bundles/org.openhab.binding.remoteopenhab/src/main/java/org/openhab/binding/remoteopenhab/internal/handler/RemoteopenhabBridgeHandler.java

```diff
@@ -324,12 +324,18 @@ public void checkConnection() {
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
                         "OH 1.x server not supported by the binding");
             } else if (getThing().getStatus() != ThingStatus.ONLINE) {
-                List<RemoteopenhabItem> items = restClient.getRemoteItems("name,type,groupType,state,stateDescription");
+                List<RemoteopenhabItem> items = restClient.getRemoteItems("name,type,groupType,stateDescription");
```

Comment:
Why would states fail but not the remaining?

Reply:
Look at the related issue. The request is failing if the response size is bigger than 2 MB. 
It is a workaround for OH3 initial release until I find 1 a way to reproduce the issue and 2 the solution to properly handle this unusual case.